x-pack/plugin/src/test/resources/rest-api-spec/test/security/hidden-index/13_security-tokens_read.yml

```diff
@@ -0,0 +1,161 @@
+---
+setup:
```

Comment:
Could we write these as java tests instead of rest ? I'm thinking ease of debugging - no other compelling reasons.

Reply:
> Could we write these as java tests instead of rest ?

For clarification, my comment was about replacing these with integration tests

Reply:
No, I don't think so. These are crucial smoke REST tests that assure us we don't leak restricted indices (for search and visibility). They don't have breadth but they test a critical feature end to end.
From my perspective, these would be the last REST tests we should be converting to Integ tests, if ever.

Reply:
Gotcha, missed the intention ( which is clearer after looking at the rest of the tests in that suite) .